source/drivers/java.txt

```diff
@@ -0,0 +1,234 @@
+.. include:: /includes/unicode-checkmark.rst
+
+===================
+Java MongoDB Driver
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+The official MongoDB Java Driver providing both synchronous and asynchronous interaction with MongoDB.
+
+- `Tutorials <http://mongodb.github.io/mongo-java-driver/3.10/driver/tutorials/>`__
+
+- `Usage Guide <http://mongodb.github.io/mongo-java-driver/3.10/driver/>`__
+
+- `API Documentation <http://mongodb.github.io/mongo-java-driver/3.10/javadoc/overview-summary.html>`_
+
+- `Changelog <http://mongodb.github.io/mongo-java-driver/3.10/whats-new/>`__
+
+- `Source Code <https://github.com/mongodb/mongo-java-driver/>`__
+
+
+MongoDB University
+------------------
+
+.. list-table::
+
+   * - .. cssclass:: bordered-figure
+       .. figure:: /figures/M220J_hero.jpg
+
+     - `M220J: MongoDB for Java Developers <https://university.mongodb.com/courses/M220J/about>`__
+ 
+       Learn the essentials of Java application development with MongoDB.
+
+
+Installation
+------------
+
+The recommended way to get started using one of the drivers in your project is with a dependency
+management system. See 
+`Installation Guide <http://mongodb.github.io/mongo-java-driver/3.10/driver/getting-started/installation/>`__
+
+
+Connect to MongoDB
+------------------
+
+Connect to a Standalone MongoDB Instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a standalone MongoDB instance:
+
+.. code-block:: java
+
+   MongoClient mongoClient = new MongoClient(new MongoClientURI("mongodb://host1:27017"));
+
+
+Connect to a Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a :manual:`replica set </replication>`, you must specify one or more members
+to the MongoClient constructor:
+
+.. code-block:: java
+
+   MongoClient mongoClient = new MongoClient(
+       new MongoClientURI("mongodb://host1:27017,host2:27017,host3:27017"));
```

Comment:
There are a few caveats to putting code examples on a drivers landing page.  The content is uncategorized and the landing page is likely to change, and connection examples are frequently bookmarked. What we might do instead is link out to the existing old page in a clear way. Or provide a link to a separate page with the connection examples on it (hopefully single-sourced somewhere and integrated with the driver's tests).

Reply:
@terakilobyte thoughts on this?